integrations.yml

```diff
@@ -275,10 +275,16 @@
   website: "https://www.lucidmeetings.com/integrations/basecamp"
   icon: "lucidmeetings.png"
   category: 6
+-
+  name: "CData Software"
+  description: "Use standards-based drivers to connect to Basecamp data in any BI tool, including Excel, Tableau, Power BI, and more!"
+  website: "https://www.cdata.com/drivers/basecamp"
+  icon: "cdata.png"
+  category: 1
 -
   name: "Akita"
   description: "Monitor the status of your Basecamp projects in Akita's powerful Customer Success Manage Platform."
   website: "https://www.akitaapp.com?utm_source=basecamp"
   icon: "akita.png"
   category: 7
-
+-
```

Comment:
@JerodJ it looks like this change leaked into your legit changes, and broke the Travis build. I imagine it was after fixing conflicts. Would you mind fixing it? I'll merge this after 😄

Reply:
@rosa fixed

Reply:
Perfect, thanks!